infra/paymentapi/src/main/resources/application.properties

```diff
@@ -26,8 +26,6 @@ payment_default_paytrail_currency=EUR
 payment_default_paytrail_language=FI
 payment_default_paytrail_return_url=http://localhost:8080/
 payment_default_paytrail_notify_url=http://localhost:8080/
-#Aggregate merchant ID
```

Comment:
this is not needed, because it is not used in the context. Only PaytrailAuthClientFactory needs the aggregate merchant id to create client for shop-in-shop use cases, but it gets the value from property `paytrail.aggregate.merchant.id`

Reply:
But can this cause errors when this is not set? Should this be set to empty value and keep key in application.properties?

Reply:
No, this property is not used at all. Only the paytrail.aggregate.merchant.id is used in PaytrailAuthClientFactory.